subprojects/gst-plugins-bad/gst-libs/gst/codecparsers/gsth264parser.c

```diff
@@ -3029,6 +3042,14 @@ gst_h264_create_sei_memory_internal (guint8 nal_prefix_size,
         payload_size_data += rud->size;
         break;
       }
+      case GST_H264_SEI_USER_DATA_UNREGISTERED:{
+        GstH264UserDataUnregistered *udu = &msg->payload.user_data_unregistered;
+
+        payload_size_data = 0;
+        payload_size_data += G_N_ELEMENTS (udu->uuid);
```

Comment:
not a requirement just a note: we are mixing styles here. In this line we prefer not to use magic number "16", and use the size of "uuid" array in GstH264UserDataUnregistered. But in the rest of the lines, such as gst_h264_write_sei_user_data_unregistered and gst_h265_parser_parse_user_data_unregistered, magic number "16" is being used.